Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1

```diff
@@ -953,6 +953,113 @@ function Parse-Exception($exception){
     return $null
 }
 
+function Get-AzureNetworkInterfaceDetails
+{
+    [CmdletBinding()]
+    param([String] [Parameter(Mandatory = $true)] $resourceGroupName,
+          [Object] [Parameter(Mandatory = $true)] $endpoint)
+
+    $accessToken = Get-SpnAccessToken $endpoint
+    $subscriptionId = $endpoint.Data.SubscriptionId.ToLower()
+
+    Write-Verbose "[Azure Rest Call] Get Resource Groups"
+    
+    $method = "GET"
+    $uri = "$($endpoint.Url)/subscriptions/$subscriptionId/resourceGroups/$resourceGroupName/providers/Microsoft.Network/networkInterfaces?api-version=2015-06-15"
```

Comment:
Looks like [NetworkInterface (List) API](https://docs.microsoft.com/en-us/rest/api/virtualnetwork/networkinterfaces/list#definitions_networkinterfacelistresult) contains NextLink attribute.
We may want to address them too.

Reply:
The current version of Azure Stack rest API doesn't support nextLink. Also, this change is because REST API being used via PS cmdlets have  version mismatch with existing API version supported by Azure Stack. So later when Azure Stack will support latest API version we can even remove this change.